app/views/admin/basic_projects/_form.html.slim

```diff
@@ -0,0 +1,46 @@
+= simple_form_for([:admin, @project],
+    html: { \
+      class: 'form-horizontal organization-record',
+    },
+    wrapper: :horizontal_form_condensed,
+    defaults: { \
+      input_html: {class: 'form-element form-control'},
+    },
+  ) do |f|
```

Comment:
Yes, @hooverlunch, we can pair on that when it is part of a sprint. Thanks for keeping an eye on that.